src/utils/developerUI/developerUI.ts

```diff
@@ -9,27 +9,50 @@ export function createDeveloperUIMonitoringPageHref(host = backend) {
     return host + '/monitoring';
 }
 
+function getDatabaseQueryString(database?: string) {
+    const params = new URLSearchParams();
+    if (database) {
+        params.set('database', database);
+    }
+    const paramsString = params.toString();
+    return paramsString ? '?' + paramsString : '';
+}
+
 // Current node connects with target node by itself using nodeId
-export const createDeveloperUILinkWithNodeId = (nodeId: number | string, host = backend) => {
+export const createDeveloperUILinkWithNodeId = (
```

Comment:
DeveloperUI links functions are sensitive to changes and easily tested, please, update tests, when you add something here